tests/cli-rustup.rs

```diff
@@ -1224,6 +1224,21 @@ fn toolchain_uninstall_is_like_uninstall() {
     });
 }
 
+#[test]
+fn toolchain_uninstall_pattern() {
+    setup(&|config| {
+        expect_ok(config, &["rustup", "uninstall", "stable-*"]);
+        expect_ok(config, &["rustup", "uninstall", "nightly-*"]);
+        let mut cmd = clitools::cmd(config, "rustup", &["show"]);
+        clitools::env(config, &mut cmd);
+        let out = cmd.output().unwrap();
+        assert!(out.status.success());
+        let stdout = String::from_utf8(out.stdout).unwrap();
+        assert!(!stdout.contains(for_host!("'stable-{}'")));
+        assert!(!stdout.contains(for_host!("'nightly-2015-01-01-{}'")));
+    });
+}
```

Comment:
I just copied this test code from other tests and modified it. Is the test correct?

Reply:
You probably want to start the test by installing and asserting that stable and nightly *are* there.  Otherwise I think this is fine.